components/setting_picture.jsx

```diff
@@ -189,7 +189,7 @@ export default class SettingPicture extends Component {
                         <li className='setting-list-item padding-top x2'>
                             <FormattedMessage
                                 id={`setting_picture.help.${imageContext}`}
-                                defaultMessage='Upload a picture in BMP, JPG, JPEG or PNG format.'
+                                defaultMessage='Upload a picture in BMP, JPG or PNG format.'
```

Comment:
@jwilander Do we no longer support JPEGs? JPEGs used to work: https://github.com/mattermost/mattermost-server/pull/6049

Reply:
JPGs and JPEGS are identical, they both worked for me when I tested my changes here. I just removed JPEG from the text because we didn't have it anywhere else and it's redundant but I'm 0/5 if we want to explicitly mention them